test/cypress/integration/consumer-tools/before-you-claim/before-you-claim.cy.js

```diff
@@ -27,4 +21,1007 @@ describe('Planning your Social Security', () => {
     cy.wait('@getSpanish');
     cy.url().should('contain', '/es');
   });
+
+  it('should display tool tip in step one after clicking svg tip icon', () => {
+    cy.get('#tooltip-container').should('not.be.visible');
+    cy.get(
+      '#step-one-form .content-l .content-l_col .input-labels span.input-labels_block span svg'
+    ).click();
+    cy.get('#tooltip-container').should('be.visible');
+  });
+
+  it('should show error if user is over the age of 70', () => {
+    claim.enterAgeOver70();
+    cy.get('#step-one-form .m-notification__warning').should('be.visible');
+  });
+
+  it('should display estimated benefits', () => {
+    claim.enterAgeUnder50();
+    claim.claimGraph().should('be.visible');
+    cy.get('#graph-container .content-l_col .learn-how a').should(
```

Comment:
Selector could be shortened to `#graph-container .learn-how a`

Another way to do this check could be:

```js
cy.get('#graph-container [href='/consumer-tools/retirement/before-you-claim/about/']').should('have.length', 1);
```